### PR TITLE
Update driver links (for 2.x) including categories!

### PR DESCRIPTION
**Pull request by tannewt**

### CODE REVIEW

File: docs/drivers.rst [hunk 1]
```diff
@@ -36,37 +36,178 @@ the ``lib/`` directory. Some drivers may not work without them.
...
+    DS1307 Real-time Clock (5V RTC Breakout) <https://circuitpython.readthedocs.io/projects/ds1307/>
+    DS3231 Real-time Clock (Precision RTC) <https://circuitpython.readthedocs.io/projects/ds3231/>
+    PCF8523 Real-time Clock (Adalogger RTC) <https://circuitpython.readthedocs.io/projects/pcf8523/>
+
+Motion Sensors
+----------------
+
+Motion relating sensing including ``acceleration``, ``magnetic``, ``gyro``, and ``orientation``.
+
+.. toctree::
```
Comment (kattni) [inline docs/drivers.rst]: No blank line after.

File: docs/drivers.rst [hunk 2]
```diff
@@ -36,37 +36,178 @@ the ``lib/`` directory. Some drivers may not work without them.
...
 
-Drivers provide easy access to sensors and other chips without requiring a
-knowledge of the interface details of the chip itself.
+These provide functionality similar to `analogio`, `digitalio`, `pulseio`, and `touchio`.
+
+.. toctree::
+
+    Adafruit SeeSaw <https://circuitpython.readthedocs.io/projects/seesaw/>
+    ADS1x15 Analog-to-Digital Converter  <https://circuitpython.readthedocs.io/projects/ads1x15/>
+    DS2413 OneWire GPIO Expander <https://circuitpython.readthedocs.io/projects/d2413/>
```
Comment (kattni) [inline docs/drivers.rst]: Is this URL incorrect?

File: docs/drivers.rst [hunk 3]
```diff
@@ -36,37 +36,178 @@ the ``lib/`` directory. Some drivers may not work without them.
...
+    FocalTech Capacitive Touch <https://circuitpython.readthedocs.io/projects/focaltouch/>
+    MCP4725 Digital-to-Analog Converter <https://circuitpython.readthedocs.io/projects/mcp4725/>
+    PCA9685 16 x 12-bit PWM Driver <https://circuitpython.readthedocs.io/projects/pca9685/>
+    TLC5947 24 x 12-bit PWM Driver <https://circuitpython.readthedocs.io/projects/tlc5947/>
+    TLC59711 12 x 16-bit PWM Driver <https://circuitpython.readthedocs.io/projects/tlc59711/>
+    MCP4725 Capacitive Touch Sensor <https://circuitpython.readthedocs.io/projects/mpr121/>
```
Comment (kattni) [inline docs/drivers.rst]: Sensor name or URL incorrect? Looks like sensor name is duplicated from above.

Comment (tannewt) [reply docs/drivers.rst]: Fixed

File: docs/drivers.rst (hunk 1 above)
Comment (tannewt) [reply docs/drivers.rst]: Good eye! Fixed.

**Comment by tannewt** [comment]
Please take another look.

**Pull request merged**